docs/website/root/manual/developer-docs/nodes/mithril-client.md

```diff
@@ -96,6 +96,36 @@ Run in debug mode with the default configuration:
 make debug
 ```
 
+## Use the mithril-client library in your own code
```

Comment:
That documentation is redundant with the new `mithril-client-library.md` that you added, should we keep it ?

Reply:
Indeed, it should be deleted, it's probably a mistake when modifying commits.

Reply:
Indeed, it should be deleted, it's probably a mistake when modifying commits.